bin/fuzzify.sh

```diff
@@ -0,0 +1,40 @@
+
+# sanity check 
+if [ $# -eq 0 ]
+  then
+    echo "No Binary name or path provided."
+    echo "To use this, simply give the program the name of the binary"
+    echo "like so: ./fuzzify ls"
+    echo "It will try to reolve it by looking in your path."
+    exit 1
+fi
+
+# get all the linked libs
+libs=$(ldd $(which $1) | grep so | sed -e '/^[^\t]/ d' | sed -e 's/\t//' | sed -e 's/.*=..//' | sed -e 's/ (0.*)//' | grep -o '^\/.*') 
```

Comment:
I think you should be able to combine those sed expressions into one sed invocation, and then just separate the statements with semicolons.

Reply:
this file actually does not need to be here

Reply:
I'll remove it